packages/jspsych/tests/data/datacollection.test.ts

```diff
@@ -31,43 +31,63 @@ describe("DataCollection", () => {
         .count()
     ).toBe(2);
   });
-  test("#ignore", () => {
+
+  test("filterColumns", () => {
+    data = [
+      { foo: "bar", rt: 100, filter: true },
+      { foo: "bar", rt: 200, filter: false },
+      { foo: "bar", rt: 300, filter: true },
+      { foo: "bar", rt: 400, filter: false },
+      { foo: "bar", rt: 500, filter: false },
+    ];
+    dataCollection = new DataCollection(data);
+
+    const filtered_data = dataCollection.filterColumns(["rt", "foo"]);
+
+    expect(
+      filtered_data.values().filter((x) => {
+        return x.foo && x.rt && !x.filter;
+      }).length
+    ).toBe(5);
```

Comment:
I think this is clearer and provides better feedback:
```suggestion
    expect(filtered_data.values()).toEqual([
      { foo: "bar", rt: 100 },
      { foo: "bar", rt: 200 },
    ]);
```